.aws/faucet.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-AMOUNT=100stake
+AMOUNTS=5stake
+#AMOUNTP=5photino
```

Comment:
uncomment or delete ?

Reply:
I wanted to keep it and uncomment for quick tests on the instance, probably when we will have a quick deploy we won't need this anymore

Reply:
uncommented in the end is healthier to check also on different denoms